### PR TITLE
[Playground] [Infrastructure] Deleting the Cloud Storage Client

### LINKED ISSUE #22720: [Playground][Task]: Updating of python scripts to save playground examples to the Cloud Datastore
### What needs to happen?

Updating python scripts to save playground examples to the Cloud Datastore during CI/CD startup

### Issue Priority

Priority: 2

### Issue Component

Component: beam-playground

### LINKED ISSUE #22865: [Playground][Task]: Deleting the Cloud Storage client
### What needs to happen?

Need to use the Cloud Datastore instead of the Cloud Storage and remove unused code related to the storage

### Issue Priority

Priority: 1

### Issue Component

Component: beam-playground

### PR DESCRIPTION
**Pull request by vchunikhin**
fixes #22865 

@olehborysevych 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: .github/workflows/playground_deploy_examples.yml [hunk 1]
```diff
@@ -133,6 +133,8 @@ jobs:
              K8S_SERVER_ADDRESS=$(kubectl get svc -n $K8S_NAMESPACE -l "app=service-python" -o jsonpath='{.items[0].status.loadBalancer.ingress[0].ip}')
              BEAM_ROOT_DIR="../../"
              BEAM_EXAMPLE_CATEGORIES="../../playground/categories.yaml"
+             GOOGLE_CLOUD_PROJECT_VALUE=${{ secrets.GCP_PLAYGROUND_PROJECT_ID }}
```
Comment (vchunikhin) [inline .github/workflows/playground_deploy_examples.yml:136]: This env variable needs to connect to the Cloud Datastore

**Comment by vchunikhin** [comment]
R: @pabloem

File: playground/backend/internal/preparers/java_preparers.go [hunk 2]
```diff
@@ -35,7 +36,7 @@ const (
 	javaPublicClassNamePattern        = "public class (.*?) [{|implements(.*)]"
 	pipelineNamePattern               = `Pipeline\s([A-z|0-9_]*)\s=\sPipeline\.create`
 	graphSavePattern                  = "String dotString = org.apache.beam.runners.core.construction.renderer.PipelineDotRenderer.toDotString(%s);\n" +
-		"    try (java.io.PrintWriter out = new java.io.PrintWriter(\"Graph.dot\")) {\n      " +
+		"    try (java.io.PrintWriter out = new java.io.PrintWriter(\"graph.dot\")) {\n      " +
```
Comment (vchunikhin) [inline playground/backend/internal/preparers/java_preparers.go:39]: This update is necessary to save graphs to cache correctly

File: playground/infrastructure/datastore_client.py [hunk 3]
```diff
@@ -106,6 +110,35 @@ def save_to_cloud_datastore(self, examples_from_rep: List[Example], sdk: Sdk):
                 self._datastore_client.delete_multi(pc_objs_keys_for_removing)
                 logging.info("Finish of deleting extra playground examples ...")
 
+    def save_catalogs(self):
```
Comment (vchunikhin) [inline playground/infrastructure/datastore_client.py:113]: This method is needed in order to load catalogs to the Cloud Datastore. It helps to decrease the coupling between python scripts and the router service

**Comment by pabloem** [comment]
can you please rebase? thanks!

**Comment by vchunikhin** [comment]
> can you please rebase? thanks!

Sure, i've merged with master

**Comment by pabloem** [comment]
lgtm thanks!

**Comment by pabloem** [comment]
Run Java PreCommit

**Comment by pabloem** [comment]
Run SQL PreCommit

**Comment by pabloem** [comment]
Run SQL_Java17 PreCommit

**Comment by pabloem** [comment]
lgtm

**Pull request merged**